### ISSUE TITLE
Doesn't respect application's BacktraceCleaner

### ISSUE THREAD

**Issue by JonCrawford** [charliesome/better_errors#308]
In Rails, the exception backtrace can be pretty noisy with a lot of stuff that doesn't add value such as your development server files. I'm used to setting up things like this when I first get started.

``` ruby
#config/initializers/backtrace_cleaner.rb
Rails.backtrace_cleaner.add_silencer { |line| line =~ /newrelic/ }
Rails.backtrace_cleaner.add_silencer { |line| line =~ /new_relic/ }
Rails.backtrace_cleaner.add_silencer { |line| line =~ /thin/ }
Rails.backtrace_cleaner.add_silencer { |line| line =~ /spring/ }
```

Since installing better_errors, these lines have returned. It seems plausible to read the silencers in the `Rails.backtrace_cleaner` object before displaying output or to provide a specific interface for filtering in better_errors.

Note: This is most important in the _server log_ since it is harder to visually scan the plaintext.

http://api.rubyonrails.org/classes/ActiveSupport/BacktraceCleaner.html

**Comment by pzgz** [comment]
Having same issue, looks like the setting made in config/initializers/backtrace_silencers.rb doesn't work if added better_errors gem. After debugging, I found removing better_errors gem fix the issue, guess the root cause was somewhere in this gem.

**Comment by RobinDaugherty** [comment]
Fixed by #343 which is included in [v2.4.0](https://github.com/charliesome/better_errors/releases/tag/v2.4.0).
